code/game/jobs/job/marines.dm

```diff
@@ -35,13 +45,9 @@
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_ADD_TO_MODE|ROLE_ADD_TO_SQUAD
 	skills_type = /datum/skills/SL
 
-	generate_wearable_equipment()
```

Comment:
Don't delete the specific equipments. Just add a `..()` at the start of it and then add to the list.

Reply:
I did that for the specialist to spawn with his specrag, but I chose to remove the backpack/satchel because marines already get one in their vendor, and sleeping with one is kinda dumb. Both on CM and DMCA people are ok to do away with them.